docs/modules/ROOT/pages/cops_rspec_rails.adoc

```diff
@@ -265,6 +265,8 @@ assert_not_includes a, b
 refute_equal(a, b)
 assert_nil a
 refute_empty(b)
+assert_true(a)
```

Comment:
The a/b naming for expected/actual, especially when used interchangeably, us getting confusing. Can we use just one for “actual”?